tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -26,8 +26,9 @@ beforeEach(() => {
 Cypress.config();
 describe('Test GitRepo Bundle name validation and max character trimming behavior in bundle', { tags: '@p1'}, () => {
   const branch = "master"
-  const path = "simple-chart"
+  const path = "qa-test-apps/nginx-app"
   const repoUrl = "https://github.com/rancher/fleet-test-data/"
+  const appName = 'nginx-keep'
```

Comment:
You exported this constant earlier on p0 test cases
If we want to do that, perhaps it would make sense to do it here as well?

Reply:
I exported all constants in the P1 test file.